main.go

```diff
@@ -145,7 +145,7 @@ func establishCustomPath(ctx *cli.Context) error {
 		if currentCtx.IsSet("work-path") && len(providedWorkPath) == 0 {
 			providedWorkPath = currentCtx.String("work-path")
 		}
-		currentCtx = currentCtx.Parent()
+		//currentCtx = currentCtx.Parent() <- FIXME
```

Comment:
```suggestion
```

Reply:
@jolheiser It had its meaning; look at: d8f28e1efbaffbee67ae7631cd99b5ec73cdfe49

Reply:
Very interesting. Okay, good check.